index.html

```diff
@@ -1,14 +1,40 @@
 <! DOCTYPE html>
 <html>
   <head>
-    <title></title>
+    <title>Salmookies Coffee</title>
+    <link href="https://fonts.googleapis.com/css?family=Permanent+Marker" rel="stylesheet">
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" type="text/css">
     <link href="css/reset.css" type="text/css" rel="stylesheet">
     <link href="css/styleindex.css" type="text/css" rel="stylesheet">
   </head>
   <body>
-    
+    <div id="salmookies"><h1>Salmookies</h1></div>
+    <img src="imgs/salmon.jpg">
+<p>If there is something we know about is <b>Salmon</b> and <b>Cookies!</b></p>
+<p>We make Salmon shaped cookies, perfect for a rainy mornig!</p>
 
+    <nav class="navbar navbar-inverse navbar-fixed-top">
+        <div class="container">
+        <div class="navbar-header">
+          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-nav-demo" aria-expanded="false">
+              <span class="sr-only">Toggle navigation</span>
+              <span class="icon-bar"></span>
+              <span class="icon-bar"></span>
+              <span class="icon-bar"></span>
+            </button>
+          <a href="#" class="navbar-brand"><span class="glyphicon glyphicon-picture" aria-hidden="true"></span> Images</a>
+        </div>
+        <div class="collapse navbar-collapse" id="bs-nav-demo">
+          <ul class=" nav navbar-nav">
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+            <li><a href="#">Locations</a></li>
+          </ul>
+        </div>
+    
 
+      <script src="http://code.jquery.com/jquery-3.3.1.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
```

Comment:
Where did these two lines come from? You're not using jQuery  or any bootstrap JS in your code.